### PR TITLE
test formatting during `check` target

### PR DESCRIPTION
**Pull request by nlhepler**
and check tests/unit as well (also format there)

### CODE REVIEW

**Comment by nlhepler** [comment]
do not merge at present!!

**Comment by nlhepler** [comment]
okay, should be ready for review. I'll rebase off master tonight/tmrw. r? @dalexander @armintoepfer

File: tests/unit/TestIntegrator.cpp [hunk 1]
```diff
@@ -360,11 +361,11 @@ TEST(IntegratorTest, TestFailAddRead)
                                     true, true)));
 }
 
-TEST(IntegratorTest, TestSucessAddRead)
+TEST(IntegratorTest, TestSuccessAddRead)
 {
     const string tpl = "AA";
     const vector<uint8_t> pw(tpl.length(), 1);
-    auto mdl = P6C4;
+    const auto mdl = P6C4;
```
Comment (armintoepfer) [inline tests/unit/TestIntegrator.cpp:368]: Ha! The casual off-target fix 😂

**Comment by armintoepfer** [comment]
I rebased!

LGTM

File: tests/unit/TestIntegrator.cpp (hunk 1 above)
Comment (nlhepler) [inline tests/unit/TestIntegrator.cpp:368]: goddammit. I thought that was lumped with the new test I wasn't going to add here. Seems I goofed the interactive add.

**Pull request merged**